src/tests/tests.spec.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable playwright/no-conditional-expect */
+/* eslint-disable playwright/no-conditional-in-test */
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+import { user } from '../test-data/users';
+
+const { login, password } = user;
+
+const testData = [
+    {
+        sortBy: 'Name (A to Z)',
+    },
+    {
+        sortBy: 'Name (Z to A)',
+    },
+    {
+        sortBy: 'Price (low to high)',
+    },
+    {
+        sortBy: 'Price (high to low)',
+    },
+];
+
+testData.forEach((data) => {
+    test.beforeEach(async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin(login, password);
+    });
+
+    test(`Verify that sorting works correctly for sorting ${data.sortBy}`, async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        const elements = (await app.inventory.inventoryItemNames.allInnerTexts()).map((el) => el.trim());
```

Comment:
Let's create separate methods `getProductsNames` and `getProductPrices` and move technical details to the `InventoryPage` page object